stock_picking_jobcan_wf_confirm/models/stock_picking_jobcan_wf_confirm.py

```diff
@@ -0,0 +1,102 @@
+import logging
+
+from odoo import _, api, models
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class StockPickingJobcanWfConfirm(models.Model):
+    _name = "stock.picking.jobcan.wf.confirm"
+    _description = "Stock Picking Jobcan Workflow Confirm"
+    _inherit = ["api.call.mixin"]
+
+    @api.model
+    def _scheduled_update(self):
+        _logger.info("Scheduled stock picking JobCan WF confirmation...")
+        pickings = self._get_assigned_pickings_with_jobcan_wf()
+        confirm_pickings = self._get_confirmed_pickings(pickings)
+
+        self._validate_confirmed_pickings(confirm_pickings)
+
+    def _get_assigned_pickings_with_jobcan_wf(self):
+        return self.env["stock.picking"].search(
+            [("state", "=", "assigned"), ("jobcan_wf_number", "!=", False)]
+        )
+
+    def _get_confirmed_pickings(self, pickings):
```

Comment:
I confused when I see the function name.
```suggestion
    def _get_jobcan_wf_confirmed_pickings(self, pickings):
```